shared/src/constants/externalAddresses.json

```diff
@@ -109,42 +105,26 @@
 	},
 	"avax": {
 		"feeds": {
-			"eth/usd": "0x0000000000000000000000000000000000000000"
+			"eth/usd": "0x0A77230d17318075983913bC2145DB16C7366156"
 		},
 		"assets": {
-			"wavax": "0x0000000000000000000000000000000000000000",
-			"rAVAX-THETA": "0x0000000000000000000000000000000000000000"
+			"wavax": "0xAc12780B07bd623dE572c4e657195F8294869664",
+			"rAVAX-THETA": "0x98d03125c62DaE2328D9d3cb32b7B969e6a87787"
 		},
-		"oTokenFactory": "0x0000000000000000000000000000000000000000",
-		"gammaController": "0x0000000000000000000000000000000000000000",
-		"gammaMarginPool": "0x0000000000000000000000000000000000000000"
+		"oTokenFactory": "0x25bC1cD298512885F0e452833198BCBb149f66a5",
+		"gammaController": "0x9e3b94819aaF6de606C4Aa844E3215725b997064",
+		"gammaMarginPool": "0xCCF6629aEaB734E621Cc59EBb0297196774fDb9D"
 	},
-	"mainnet-sim": {
-		"assets": {
-			"eth": "0x0000000000000000000000000000000000000000",
-			"weth": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
-			"wbtc": "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599",
-			"usdc": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"
-		}
-	},
-	"avax": {
+	"fuji": {
 		"feeds": {
-			"eth/usd": "0x0000000000000000000000000000000000000000"
+			"eth/usd": "0xD9D01A9F7C810EC035C0e42cB9E80Ef44D7f8692"
 		},
 		"assets": {
-			"wavax": "0x0000000000000000000000000000000000000000",
-			"rAVAX-THETA": "0x0000000000000000000000000000000000000000"
+			"wavax": "0xD9D01A9F7C810EC035C0e42cB9E80Ef44D7f8692",
```

Comment:
wavax is wrong here? should be `0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7`

Reply:
This is right for the testnet fuji but the mainnet address is wrong.  Fixing.